app/src/main/java/com/example/happymeals/adapters/RecipeStorageAdapter.java

```diff
@@ -63,15 +63,17 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
         servings.setText( String.valueOf( currentRecipe.getServings() ) );
         description.setText( currentRecipe.getDescription() );
 
+        /*
         listItem.setOnClickListener(new View.OnClickListener() {
```

Comment:
Why is this commented out? Does it affect anything else in the code being removed?